CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ index download_links: ->{ can?(:view_all_download_links) || [:pdf] }
 ```
 * Comments menu can be customized via configuration passed to `config.comments_menu` [#4187][] by [@drn][]
 * Added `config.route_options` to namespace to customize routes [#4467][] by [@stereoscott[]]
+* Allow custom panel title given with `attributes_table` [#4940][] by [@ajw725][]
```

Comment:
You need to add it at the top in a new `## Master (unreleased)` section since this is the first change in master after 1.0.0 release. Sorry for not mentioning it!